src/demo-app/mock-data/models.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  HDF5Id,
+  HDF5Collection,
+  HDF5Group,
+  HDF5Dataset,
+  HDF5Datatype,
+} from '../../h5web/providers/models';
+
+export interface MockHDF5Metadata {
```

Comment:
This interface is specific to mocked JSON data, so I move into the same folder and rename it.